core/kits/documents/kit.php

```diff
@@ -51,6 +51,17 @@ public function get_tabs() {
 		return $this->tabs;
 	}
 
+	/**
+	 * Retrieve a tab by ID.
+	 *
+	 * @return Tabs\Tab_Base
+	 */
+	public function get_tab( $id ) {
+		$tabs = $this->get_tabs();
+
+		return isset( $tabs[ $id ] ) ? $tabs[ $id ] : null;
```

Comment:
You can simply do `return self::get_items( $this->get_tabs(), $id );`

Reply:
Done.